sdk/python/lib/pulumi/automation/_stack.py

```diff
@@ -211,6 +211,8 @@ def up(
         on_output: Optional[OnOutput] = None,
         on_event: Optional[OnEvent] = None,
         program: Optional[PulumiFn] = None,
+        verbose: Optional[int] = None,
+        log_to_stderr: Optional[bool] = None,
```

Comment:
Does this need to be `Optional[bool]`, can it not just be a `bool` with default `False`>